pax/simulation.py

```diff
@@ -465,33 +468,31 @@ def make_pax_event(self):
 
             # Did you want to superpose onto real noise samples?
             if self.config['real_noise_file']:
-                if noise_sample_mode != 'coherent':
+                #if noise_sample_mode != 'coherent':
                     # For each channel, choose different noise sample numbers
-                    chosen_noise_sample_numbers = np.random.randint(0,
+                    #chosen_noise_sample_numbers = np.random.randint(0,
                                                                     available_noise_samples - 1,
                                                                     needed_noise_samples)
 
-                    roll_number = np.random.randint(noise_sample_len)
+                    #roll_number = np.random.randint(noise_sample_len)
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->